lectures/geom_series.md

```diff
@@ -917,7 +951,32 @@ plt.show()
 Increasing the marginal propensity to consume $b$ increases the
 path of output over time.
 
-Now we will compare the effects on output of increases in investment and government spending.
+```{solution-end}
+```
+
+
+```{exercise-start}
+:label: geom_ex3
+```
+Continue from {ref}`geom_ex1` with the following values
+
+```{code-cell} ipython3
+values = [0.3, 0.4]
+```
+
+First, plot the time paths of $y_t$ using the same initial valuesm but with $i$ 
```

Comment:
@SylviaZhaooo can you please run a spell check. `valuesm` should be `values`

Reply:
No problem.